include/unifex/adapt_stream.hpp

```diff
@@ -23,57 +23,61 @@
 namespace unifex {
 
 template <
-    typename StreamSender,
+    typename Stream,
     typename NextAdaptFunc,
     typename CleanupAdaptFunc>
 struct adapted_stream {
-  StreamSender innerStream_;
+  Stream innerStream_;
   NextAdaptFunc nextAdapter_;
   CleanupAdaptFunc cleanupAdapter_;
 
-  auto next() {
-    return std::invoke(nextAdapter_, cpo::next(innerStream_));
+  friend auto tag_invoke(tag_t<next>, adapted_stream& s)
```

Comment:
thoughts for the future - perhaps if there was a way to get clang format to always add a <CR> before the first arg in a tag_invoke decl/def?

Reply:
To always add a what?